universidade-estadual-do-oeste-do-parana-programa-institucional-de-bolsas-de-iniciacao-cientifica.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Universidade Estadual do Oeste do Paraná - Programa Institucional de Bolsas de Iniciação Científica (Portuguese - Brazil)</title>
+    <title-short>PIBIC</title-short>
+    <id>http://www.zotero.org/styles/universidade-estadual-do-oeste-do-parana-programa-institucional-de-bolsas-de-iniciacao-científica-portuguese-brazil</id>
```

Comment:
Almost there.

This needs to end on "universidade-estadual-do-oeste-do-parana-programa-institucional-de-bolsas-de-iniciacao-cientifica" (no accent in científica, and without the "-portuguese-brazil")

Same for the "self" link.